sigllm/data.py

```diff
@@ -34,30 +68,12 @@ def sig2str(values, sep=',', space=False, decimal=0):
     values = np.abs(values)
 
     sequence = sign * (values * 10**decimal).astype(int)
+    
+    #Rescale all elements to be nonnegative
+    sequence = sequence - min(sequence)
```

Comment:
I'm wondering if rescaling should be a parameter to this function. I feel like we can test if keeping negative values is fine and we can `rescale` it to be nonnegative if indicated.

```python3
if rescale:
    # rescale elements to be nonnegative
    sequence = sequence - min(sequence)
```

`rescale=True` by default